setup.py

```diff
@@ -56,7 +56,7 @@ def _prepare_extras():
     include_package_data=True,
     zip_safe=False,
     keywords=["deep learning", "pytorch", "AI"],
-    python_requires=">=3.7",
+    python_requires=">=3.8",
```

Comment:
Why was this needed? 3.7 still has eight months of life left

Reply:
ok :)

Reply:
on the other hand py3.10 does not work :D